ansible/roles/geoblacklight/tasks/geoblacklight_setup.yml

```diff
@@ -16,19 +16,13 @@
     force: yes
   become_user: "{{ project_user }}"
 
-- name: check for secrets file
-  stat:
-    path: "{{ project_app_root }}/config/secrets.yml"
-  register: secret
```

Comment:
The original code was something left over from the version that Francis started.  I don't believe `secret` was ever actually used.  It is not needed now, at least.